### PR TITLE
DE-253 Add basic configuration for test coverage greater than 80%

### PR DESCRIPTION
**Pull request by paangaflo**
feat: [DE-253](https://makingsense.atlassian.net/browse/DE-253).

Execution in CI environment with failure:
image

Execution in CI environment with coverage success full:
image

Execution in local environment with failure:
`yarn coverage:local`
image

Execution in local environment with coverage success full:
`yarn coverage:local`
image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,6 +17,10 @@ FROM restore AS test
 ENV CI=true
 RUN yarn test
 
+FROM test AS coverage
+ENV CI=true
```
Comment (andresmoschini) [inline Dockerfile:21]: La variable de entorno no es necesaria porque ya está definido en la etapa de `test`

```suggestion
FROM test AS coverage
```

File: Dockerfile [hunk 2]
```diff
@@ -17,6 +17,10 @@ FROM restore AS test
...
+RUN yarn coverage:ci
```
Comment (andresmoschini) [inline Dockerfile:22]:
```
RUN yarn test 
    → react-scripts test
RUN yarn coverage:ci
    → react-scripts test --coverage",
```

Es necesario ejecutar los test dos veces? no alcanza con _coverage_?

**Comment by andresmoschini** [comment]
🚀

File: Dockerfile (hunk 1 above)
Comment (paangaflo) [reply Dockerfile]: Ok. Changed.

File: Dockerfile (hunk 2 above)
Comment (paangaflo) [reply Dockerfile]: Ok. Changed.

**Pull request merged**